ui-v2/lib/block-slots/addon/components/block-slot.js

```diff
@@ -1,3 +1,4 @@
+import { readOnly } from '@ember/object/computed';
 import Component from '@ember/component';
 import { isPresent } from '@ember/utils';
 import { get, set, computed, defineProperty } from '@ember/object';
```

Comment:
It seems like `computed` is no longer used? Also, `get` is still present on line 42, but it could probably be removed.

Reply:
Hmmm yeah this one is really weird..

2 bits of weirdness:

A codemod added the `{ readOnly }`, but it seems like the `get` codemod missed/skipped the `get` here. I'm just double checking to make sure there's no reason it missed/skipped it before changing it manually.

---

I've also noticed that ember/testem eslint tests don't seem to include any files in `lib`. (my custom linter does pick up on this though, which uses the same .eslintrc file)

Do you know if this is default ember configuration? And if so any idea where I can make sure ember includes any files in `lib` during testem testing?

Reply:
hmm well test-loading _used_ to happen with [this addon](https://github.com/ember-cli/ember-cli-test-loader) I believe, but it seems like it might not be used anymore. I don’t know how to get it to run tests outside `tests/`, I guess I always follow the conventions in this way.

Reply:
I took another brief look at this, but couldn't figure out how I can get these files included into my ember/testem test suite, so gave up. I do have a `make lint` Makefile target that runs a linter on all of these additional files (actually I could just add this before my `make test`, not in this PR though). This is a manual thing, but as its a linter, I'm ok with that.

Once I'd given up on looking into this further, I also manually updated the things the codemod didn't get in this file.